src/StoreKit/SKCloudServiceSetupOptions.cs

```diff
@@ -10,10 +12,12 @@ partial class SKCloudServiceSetupOptions {
 
 		public virtual SKCloudServiceSetupAction? Action {
 			get {
-				return (SKCloudServiceSetupAction?) (SKCloudServiceSetupActionExtensions.GetValue (_Action));
+				if (_Action is null)
+					ObjCRuntime.ThrowHelper.ThrowArgumentNullException (nameof (_Action));
+				return (SKCloudServiceSetupAction?) (SKCloudServiceSetupActionExtensions.GetValue (_Action!));
```

Comment:
I know that we should not throw an exception in a getter, but the `SKCloudServiceSetupActionExtensions.GetValue` throws an exception if the _Action is null anyways... Suggestion?